jupyter-book/reproducibility/reproducibility.ipynb

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
<p>Well, these should come at the end of the chapter :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>